src/context/Global.tsx

```diff
@@ -110,6 +112,14 @@ const GlobalProvider = (props: { children: any }) => {
         name: "denomination",
         ...stringSerializer,
     });
+    const localeSeparator = (0.1).toLocaleString().charAt(1);
+    const [separator, setSeparator] = makePersisted(
```

Comment:
Why persist this?

Reply:
i thought when you change it once  you going to expect it the next time.